### PR TITLE
Make search bar

### PR DESCRIPTION
**Pull request by glebdronchenko**

### CODE REVIEW

File: src/components/Header/components/LocationSearch/LocationSearch.jsx [hunk 1]
```diff
@@ -1,14 +1,65 @@
...
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentLocation: 'Gomel',
+      isOpenDropDown: false
+    };
+  }
+
+  dropdownContainer = React.createRef();
```
Comment (anton-vlasik) [inline src/components/Header/components/LocationSearch/LocationSearch.jsx]: either declare here all instance properties or in the constuctor

File: src/components/Header/components/LocationSearch/components/SearchedLocations/SearchedLocations.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import classes from './SearchedLocations.module.scss';
+import SearchedLocation from '../SearchedLocation/SearchedLocation';
+import ApiService from '../../../../../../services/ForecastApiService';
+
+import { useState, useEffect } from 'react';
+import { useCookies } from 'react-cookie';
+
+function SearchedLocations({ setChangeLocation, searchString, ...props }) {
```
Comment (anton-vlasik) [inline src/components/Header/components/LocationSearch/components/SearchedLocations/SearchedLocations.jsx]: it seems to me that `await getSearchLocations(searchString, cookies);` should happen above in SearchDropDown. You are only passing down searchString in order to fetch something with it's help here. Are you sure it is a right place to do that ?

File: src/components/Header/components/LocationSearch/LocationSearch.jsx [hunk 3]
```diff
@@ -1,14 +1,65 @@
...
+
+  componentDidMount() {
+    document.addEventListener('mousedown', this.handleClickOutsideDropdown);
```
Comment (anton-vlasik) [inline src/components/Header/components/LocationSearch/LocationSearch.jsx]: would this event fire if you click right above your dropdown?

Comment (glebdronchenko) [reply src/components/Header/components/LocationSearch/LocationSearch.jsx]: No, but already I have fixed it. Problem was in wrappers.

File: src/components/Header/components/LocationSearch/components/SearchedLocations/SearchedLocations.jsx (hunk 2 above)
Comment (glebdronchenko) [reply src/components/Header/components/LocationSearch/components/SearchedLocations/SearchedLocations.jsx]: Done. I have moved it to _SearchDropDown_ component. But maybe I need to move it into another file?

**Review by anton-vlasik — APPROVED** [review]

**Review by ximet — APPROVED** [review]

**Pull request merged**